spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -205,6 +209,20 @@
         ).to be >= 0.0
       end
 
+      context 'with a request delay in ms' do
+        let(:request_delay_ms) { 750 }
+
+        it 'adds a delay between requests to USPS' do
+          allow(InPersonEnrollment).to receive(:needs_usps_status_check).
+            and_return(pending_enrollments)
+          stub_request_passed_proofing_results
+          expect(job).to receive(:sleep).exactly(pending_enrollments.length - 1).times.
```

Comment:
do we maybe want to throw an `allow(job).to receive(:sleep)` on all the other specs so we don't risk actually slowing down the tests?

Reply:
I just went the route of [stubbing the config value to 0](https://github.com/18F/identity-idp/pull/6959/files#diff-e13dfd4773e4aaec3f5a236d4dab1ba59c0ae6eee062da49bb7970993419c597R83-R93) for the other specs. Lmk if that seems insufficient and I can add your suggestion -- stubbing the value to 0 seems sufficient to me

Reply:
oh that is great too! I missed that in the diff